frontend/src/models/region.ts

```diff
@@ -22,9 +22,9 @@ const COUNTRY_LIST = [
     "VN", "VU", "WF", "WS", "YE", "YT", "ZA", "ZM", "ZW",
 ] as const;
 
-export type Country = typeof COUNTRY_LIST[number] | "";
+export type Region = typeof COUNTRY_LIST[number] | "";
```

Comment:
Probably should change `COUNTRY_LIST` to `REGION_LIST` for full coverage.

Reply:
fixed